spec/fixtures/custom_nginx.template

```diff
@@ -312,60 +312,156 @@ http {
+            }
             content_by_lua_block {
-                local server = require "resty.websocket.server"
-                local wb, err = server:new{
-                  timeout = 5000,
-                  max_payload_len = 65535,
-                }
-
-                if not wb then
-                  ngx.log(ngx.ERR, "failed to open websocket: ", err)
-                  return ngx.exit(444)
-                end
+                local mu = require('spec.fixtures.mock_upstream')
+                return mu.send_json_response({
+                    valid_routes = {
+                        ["/ws"] = "web sockets echo server",
+                        ["/get"] = "Accepts a GET request and returns it in json format",
+                        ["/post"] = "Accepts a POST request and returns it in json format",
+                        ["/anything"] = "Accepts any request and returns it in json format",
+                        ["/request"] = "Alias to /anything",
+                        ["/delay/s"] = "Delay the response for s seconds",
+                        ["/basic-auth/usr/pass"] = "Performs basic authentication with the given credentials",
+                        ["/status/code"] = "Returns a response with the specified status code"
+                    }
+                })
+            }
+        }
 
-                while true do
-                  local data, typ, err = wb:recv_frame()
-                  if wb.fatal then
-                    ngx.log(ngx.ERR, "failed to receive frame: ", err)
-                    return ngx.exit(444)
-                  end
+        location = /ws {
+            content_by_lua_block {
+                local mu = require('spec.fixtures.mock_upstream')
+                return mu.serve_web_sockets()
+            }
+        }
 
-                  if data then
-                    if typ == "close" then
-                      break
-                    end
-
-                    if typ == "ping" then
-                      local bytes, err = wb:send_pong(data)
-                      if not bytes then
-                        ngx.log(ngx.ERR, "failed to send pong: ", err)
-                        return ngx.exit(444)
-                      end
-
-                    elseif typ == "pong" then
-                      ngx.log(ngx.INFO, "client ponged")
-
-                    elseif typ == "text" then
-                      local bytes, err = wb:send_text(data)
-                      if not bytes then
-                        ngx.log(ngx.ERR, "failed to send text: ", err)
-                        return ngx.exit(444)
-                      end
-                    end
+        location = /get {
+            access_by_lua_block {
+                local mu = require('spec.fixtures.mock_upstream')
+                return mu.filter_access_by_method("GET")
+            }
+            content_by_lua_block {
+                local mu = require('spec.fixtures.mock_upstream')
```

Comment:
Could we stay consistent and only use double quotes in those requires? It could be that single quotes are used in the Lua handlers of this file (I haven't checked), but if so, it's a mistake...